views/lti/editorMethods.jade

```diff
@@ -28,7 +28,7 @@ script
   :babel
     function initializeCKEditor(isBmc = false) {
         CKEDITOR.on('instanceCreated', function (e) {
-          e.editor.on('change', autosave);
+          e.editor.on('focus', autosave);
```

Comment:
Make sure to unbind the event on blur or we will have a new event listener registered every time a user clicks on the editor.